### PR TITLE
Add tandem-genotypes

### PR DESCRIPTION
**Pull request by mcfrith**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

I'm new to this, probably doing some things wrong...

### CODE REVIEW

File: recipes/tandem-genotypes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+package:
+  name: tandem-genotypes
+  version: {{ version }}
+
+source:
+  url: https://github.com/mcfrith/tandem-genotypes/archive/{{ version }}.tar.gz
+  sha256: a5a48ca33a2f11bdee4fcfa8dc2da4306b958b141b40014c1a0362a7bdd836bd
+
+build:
```
Comment (corburn) [inline recipes/tandem-genotypes/meta.yaml]: add `noarch: python`
https://bioconda.github.io/linting.html#should-be-noarch

File: recipes/tandem-genotypes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
```
Comment (corburn) [inline recipes/tandem-genotypes/meta.yaml]:
```
host:
  - python
```

File: recipes/tandem-genotypes/meta.yaml (hunk 1 above)
Comment (mcfrith) [reply recipes/tandem-genotypes/meta.yaml]: Many thanks for looking at this.

But your link says: "Don’t do this if your Python package has a command line interface", which it does.

Also this:
https://bioconda.github.io/guidelines.html#id3

I'm confused...

Comment (corburn) [reply recipes/tandem-genotypes/meta.yaml]: @mcfrith the documentation is lagging behind a recent update of the bioconda recipes and build system to conda3.

**Comment by corburn** [comment]
Adding `noarch` resolved the lint warning:

Adding a host section should resolve the `bin/sh: tandem-genotypes: not found` error.

```
requirements:
  host:
    - python
    - setuptools
```

**Comment by mcfrith** [comment]
Many thanks for your help!

**Pull request merged**